src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -286,6 +291,8 @@ class OTARequestor : public OTARequestorInterface, public BDXDownloader::StateDe
     OTAUpdateStateEnum mCurrentUpdateState = OTAUpdateStateEnum::kIdle;
     Server * mServer                       = nullptr;
     chip::Optional<bool> mRequestorCanConsent;
+    BitMapObjectPool<ProviderLocationType, CHIP_CONFIG_MAX_FABRICS> mDefaultOtaProviderList;
```

Comment:
Drive-by request to use `ObjectPool` (identical to `BitMapObjectPool` on small platforms, uses heap on big platforms, which gives better ASAN coverage).